src/content/docs/Language Fundamentals/naming.md

```diff
@@ -2,18 +2,18 @@
 title: Naming Rules
 description: Naming Rules
 sidebar:
-    order: 129
+    order: 41
 ---
 
-C3 introduces fairly strict naming rules to reduce ambiguity. 
+C3 introduces fairly strict naming rules to reduce ambiguity, make the language compile faster and simplify the compiler. 
```

Comment:
Change to:
C3 introduces fairly rigid naming rules to reduce ambiguity and make the language easy to parse for tools.